src/SprykerEco/Yves/Easycredit/Theme/default/components/molecules/easycredit-summary/easycredit-summary.twig

```diff
@@ -0,0 +1,25 @@
+{% extends model('component') %}
+
+{% define config = {
+    name: 'easycredit-summary',
+    tag: 'div'
+} %}
+
+{% define data = {
+    title: 'EasyCredit Summary:',
```

Comment:
Is it ok it's not translated?

Reply:
we have no acceptance criteria for this part and no specs, so it just placeholder for now

Reply:
But you aligned it with TL?